src/app.js

```diff
@@ -1,11 +1,16 @@
+
 import express from 'express';
-import auth from './routes/auth.route';
+import morgan from 'morgan';
+import auth from './routes/auth/auth.route';
+import protect from './routes/protected.route';
```

Comment:
Import name must match both the file name and the exported name. E.g if the file name is `auth.route.js`, the import and export name must be `authRoute`. This should be corrected in other files as well.